sunpy/io/tests/test_jp2.py

```diff
@@ -36,3 +36,12 @@ def test_read_file():
     """
     map_ = Map(AIA_193_JP2)
     assert isinstance(map_, GenericMap)
+
+
+@skip_glymur
+def test_read_memmap():
+    """
+    Passing memmap as keyword argument should not raise an error.
+    """
+    map_ = Map(AIA_193_JP2, memmap=True)
+    assert isinstance(map_, GenericMap)
```

Comment:
Do we not want to check the data is memory mapped?

Reply:
Only the FITS reader supports the `memmap` kwarg, so whether the data is actually memory mapped or not doesn't depend on the value of `memmap` passed.

Reply:
Right, I for some reason though this was a fits file.